<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>440</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>440</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/images/XHS_166000877449601027901kn4vl9wgmd1010jnc0h1bnbb.jpg);
</string>
  </property>
  <widget class="QWidget" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>0</y>
     <width>421</width>
     <height>441</height>
    </rect>
   </property>
   <widget class="QPushButton" name="btn_login">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>400</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 127);</string>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>290</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="widget_3" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>用户名：  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledit_username"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_4" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>密码：    </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledit_pwd"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_6" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>昵称：    </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledit_name"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>性别：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbtn_male">
          <property name="text">
           <string>男</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbtn_female">
          <property name="text">
           <string>女</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_7" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>年龄：    </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledit_age"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>104</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>欢迎注册</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
